cob_bringup/components/cam3d_zr300_rgbd.launch

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<launch>
+	<arg name="robot"/>
+	<arg name="name"/>
+	<arg name="num_worker_threads" default="4"/>
+	<arg name="start_manager" default="true"/>
+	<arg name="flip" default="true"/>
+	<arg name="sim" default="false"/>
+	<arg name="serial_no" default=""/>
+
+	<include unless="$(arg sim)" file="$(find realsense_camera)/launch/zr300_nodelet_rgbd.launch">
```

Comment:
If we like to release cob_robots we should fall back to the typo rgdb, so we can use the released realsense package.